npm/nameSpaceController.go

```diff
@@ -199,35 +179,21 @@ func (nsc *nameSpaceController) deleteNamespace(obj interface{}) {
 		return
 	}
 
-	nsc.npMgr.Lock()
```

Comment:
We should filter events in these functions itself. reducing the burden on worker go routine.

Reply:
same with above comment.